### PR TITLE
v2.0 readme updates

### PR DESCRIPTION
**Pull request by kevingreenman**
## Description
Update README and other non-code files for the release of v2.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,58 @@
...
+
+- Documentation of Chemprop v1 is available [here](https://chemprop.readthedocs.io/en/v1.7.1/). Note that the content of this site is several versions behind the final v1 release (v1.7.1) and does not cover the full scope of features available in chemprop v1.
+- The v1 [README](https://github.com/chemprop/chemprop/blob/v1.7.1/README.md) is the best source for documentation on more recently-added features.
+- Please also see descriptions of all the possible command line arguments in the v1 [`args.py`](https://github.com/chemprop/chemprop/blob/v1.7.1/chemprop/args.py) file.
+
+### v1 Tutorials and Examples
+
+- [Benchmark scripts](https://github.com/chemprop/chemprop_benchmark) - scripts from our 2023 paper, providing examples of many features using Chemprop v1.6.1
+- [ACS Fall 2023 Workshop](https://github.com/chemprop/chemprop-workshop-acs-fall2023) - presentation, interactive demo, exercises on Google Colab with solution key
+- [Google Colab notebook](https://colab.research.google.com/github/chemprop/chemprop/blob/master/colab_demo.ipynb) - several examples, intended to be run in Google Colab rather than as a Jupyter notebook on your local machine
```
Comment (shihchengli) [inline README.md:51]: Will the URL be changed?

Comment (kevingreenman) [reply README.md]: good catch, yes, the colab notebook link will change. The others should all be updated already or staying the same

File: CONTRIBUTING.md
Comment (KnathanM) [inline CONTRIBUTING.md:1]: Suggested change:
`When you're ready to be considered for merging, check the "Ready to go"` -> `When you're ready to be considered for merging, click the "Ready for review"`

Remove `* [Thinkful PR example](http://www.thinkful.com/learn/github-pull-request-tutorial/#Time-to-Submit-Your-First-PR)
` as the link is broken

File: README.md [hunk 2]
```diff
@@ -1,11 +1,58 @@
...
+There are tutorial notebooks in the [`examples/`](https://github.com/chemprop/chemprop/tree/main/examples) directory.
 
-You can subscribe to our development status and notes for this version [here](https://github.com/chemprop/chemprop/issues/517).
+Chemprop recently underwent a ground-up rewrite and new major release (v2.0.0). A helpful transition guide from Chemprop v1 to v2 can be found [here](https://docs.google.com/spreadsheets/u/3/d/e/2PACX-1vRshySIknVBBsTs5P18jL4WeqisxDAnDE5VRnzxqYEhYrMe4GLS17w5KeKPw9sged6TmmPZ4eEZSTIy/pubhtml). This includes a side-by-side comparison of CLI argument options, a list of which arguments will be implemented in later versions of v2, and a list of changes to default hyperparameters.
+
+**License:** Chemprop is free to use under the [MIT License](LICENSE.txt). The Chemprop logo is free to use under [CC0 1.0](docs/source/_static/images/logo/LICENSE.txt).
+
+**References**: Please cite the appropriate papers if Chemprop is helpful to your research.
```
Comment (KnathanM) [inline README.md:24]: Should we note that all these references were for v1? My main concern is the interpretation functionality line, given we don't have that in v2 yet.

File: CONTRIBUTING.md
Comment (kevingreenman) [reply CONTRIBUTING.md:1]: good catches, done

File: README.md (hunk 2 above)
Comment (kevingreenman) [reply README.md:24]: good point, I've clarified this

**Review by KnathanM — APPROVED** [review]

**Pull request merged**